Tests/New-BBServerTag.Tests.ps1

```diff
@@ -0,0 +1,209 @@
+﻿# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+#     http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+Set-StrictMode -Version 'Latest'
+#Requires -Version 4
+& (Join-Path -Path $PSScriptRoot -ChildPath 'Initialize-BitbucketServerAutomationTest.ps1' -Resolve)
+
+function New-BBServerTestConnection

```

Comment:
Did you take this from Tests\BitbucketServerAutomation\BitbucketServerAutomation.psm1? If so, put it back there.